dist/streams.js

```diff
@@ -155,6 +160,7 @@ class MetacomWritable extends EventEmitter {
     this.streamId = initData.streamId;
     this.name = initData.name;
     this.size = initData.size;
+    this.chunk = null;
```

Comment:
Maybe it's better to hold chunks in array before concatenation and concatenate them once.

Reply:
Done